### PR TITLE
added () to config variant trimming character set

### PR DESCRIPTION
**Pull request by DavidWoohyunLee**

### CODE REVIEW

**Review by yonaskolb — COMMENTED** [review]
Thanks @DavidWoohyunLee! Could you please:
- add a changelog entry under `Next Version` -> `Fixed`
- add some test cases here https://github.com/yonaskolb/XcodeGen/blob/master/Tests/ProjectSpecTests/ProjectSpecTests.swift#L361

**Comment by DavidWoohyunLee** [comment]
Hi @yonaskolb , I added some test cases and updated the changelog!

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +16,7 @@
 #### Fixed
 - Fixed no such module `DOT` error when package is used as a dependency [#1067](https://github.com/yonaskolb/XcodeGen/pull/1067) @yanamura
 - Fixed scheme config variant lookups for some configs like `ProdDebug` and `Prod-Debug` that broke in 2.21.0 [#1070](https://github.com/yonaskolb/XcodeGen/pull/1070) @yonaskolb
+- Fixed scheme config variant lookups for some configs like ``
```
Comment (yonaskolb) [inline CHANGELOG.md:19]:
```suggestion
```

**Comment by DavidWoohyunLee** [comment]
@yonaskolb removed that extra line.

**Comment by DavidWoohyunLee** [comment]
@yonaskolb Fixed the incorrect test case

**Comment by DavidWoohyunLee** [comment]
@yonaskolb I don't know why it's failing with fatal error.. I created a new pull request. https://github.com/yonaskolb/XcodeGen/pull/1080

**Comment by yonaskolb** [comment]
@DavidWoohyunLee it's ok, it's an XcodeProj error. Linux builds have started failing on Github Actions. I've opened a PR here https://github.com/tuist/XcodeProj/pull/615

**Pull request merged**